src/javascript/binary/static/common/common.utility.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Created by qingwei on 10/13/15.
```

Comment:
no need for "Created by qingwei on 10/13/15" comment; we can find out who wrote what with the 'git blame' function.